### PR TITLE
Chore: add changelog for 0.4.0

### PR DESCRIPTION
**Pull request by fgksgf**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,20 @@ Changes by Version
...
 
+0.4.0
+------------------
+
+### Features
+
+- Add `dashboard global` command with auto-refresh
+- Add `dashboard global-metrics` command
+- Add traces search
+- Refactor `metrics thermodynamic` command to adopt the new query protocol
```
Comment (wu-sheng) [inline CHANGES.md:13]: Is this for adopting v2 query protocol?

Comment (fgksgf) [reply CHANGES.md:13]: yes

**Review by kezhenxu94 — APPROVED** [review]

File: CHANGES.md (hunk 1 above)
Comment (wu-sheng) [reply CHANGES.md:13]: Is the metrics query(besides thermodynamic) adopting the v2 already?

Comment (fgksgf) [reply CHANGES.md:13]: no, will do that in the next version.

**Pull request merged**